CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # [Changelog](https://github.com/yola/drf-madprops)
 
-## dev
+## 2.0.0
 * Dropped Python 2.6 support
+* Added Python 3 support
+* Add running tests Python 3.5-3.9, Django 1.11-3.2, djangorestframework 3.5-3.12
```

Comment:
Add -> Added

Reply:
btw, do we need this info in the changelog?

Reply:
Yeah, good point - probably we don't